src/rpc/quorums.cpp

```diff
@@ -246,7 +246,27 @@ static RPCHelpMan quorum_info()
         RPCResult{
             RPCResult::Type::OBJ, "", "",
             {
-                {RPCResult::Type::ELISION, "", ""}
+                {RPCResult::Type::NUM, "height", "Quorum Height"},
+                {RPCResult::Type::STR, "type", "Quorum type"},
+                {RPCResult::Type::STR_HEX, "quorumHash", "Quorum hash"},
+                {RPCResult::Type::NUM, "quorumIndex", "Quorum index (applicable only to rotated quorums)."},
```

Comment:
`quorumIndex` is defined twice (see below), consider adding it to `RPCRESULT_MAP`

https://github.com/dashpay/dash/blob/f650f655879f951537000ab94f7cca40752bec93/src/rpc/quorums.cpp#L116

https://github.com/dashpay/dash/blob/f650f655879f951537000ab94f7cca40752bec93/src/rpc/quorums.cpp#L299